samples/rest/BingVideoSearchV7.py

```diff
@@ -7,36 +7,45 @@
 import os
 from pprint import pprint
 import requests
+from dotenv import load_dotenv
+
+# Load the environment variables from .env file
+load_dotenv()
 
 '''
 This sample makes a call to the Bing Video Search API with a topic query and returns relevant video with data.
 Documentation: https://docs.microsoft.com/en-us/bing/search-apis/bing-video-search/overview
 '''
 
+AUTH_HEADER=os.environ.get('BING_SEARCH_V7_AUTHORIZATION_HEADER', 'Ocp-Apim-Subscription-Key')
+API_KEY='BING_SEARCH_V7_VIDEO_SEARCH_SUBSCRIPTION_KEY'
+
 # Add your Bing Search V7 subscription key and endpoint to your environment variables.
-subscriptionKey = os.environ['BING_SEARCH_V7_SUBSCRIPTION_KEY']
-endpoint = os.environ['BING_SEARCH_V7_ENDPOINT'] + "/bing/v7.0/videos/search"
+endpoint = os.environ.get('BING_SEARCH_V7_VIDEO_SEARCH_ENDPOINT', 'https://api.bing.microsoft.com/v7.0/videos')
+subscriptionKey = os.environ.get(API_KEY)
+if subscriptionKey is None:
+    raise(RuntimeError(f'Please define the {API_KEY} environment variable'))
 
 # Search query
-query = "kittens"
+query = 'kentucky derby'
 
 # Construct a request
 headers = {
     'Content-Type': 'application/json',
-    'Ocp-Apim-Subscription-Key': subscriptionKey
+    AUTH_HEADER: subscriptionKey
     }
-params = { "q": query }
+params = { 'q': query }
 
 # Call the API
 try:
-    response = requests.get(endpoint, headers=headers, params=params)
+    response = requests.get(endpoint + '/search', headers=headers, params=params)
```

Comment:
Related to my other comment about the customizability of the endpoint -- I think that given the features available today that either the endpoint is fully hard-coded, is fully-settable by environment variable and does not have a hard-coded /search component, or only really the hostname settable and that's the same environment variable among all the API samples.

Reply:
The video search (and some others) actually has 3 endpoints, `/videos/search`, `/videos/trending`, and `/videos/details`. So I figured I add one sample for each endpoint that's why I hardcoded the last part. But after seeing your comments, I'm leaning towards either hardcoding the endpoints...